webcompat/static/css/development/components/form.css

```diff
@@ -98,7 +98,9 @@
 }
 
 /* state */
-.wc-Form-item.is-hidden {
+.wc-Form-item.is-hidden,
+.js-edit-img-open-btn.is-hidden,
```

Comment:
for the .is-hidden class, I agree with you the selectors should be simpler but the current pattern across the css is that it is combined with other selectors. do you mind if I update here (form.css) and open a separate issue to fix the other selectors in remaining *.css?